pkgs/tools/security/kubescape/default.nix

```diff
@@ -6,7 +6,7 @@
 
 buildGoModule rec {
   pname = "kubescape";
-  version = "2.0.156";
+  version = "2.0.158";
 
   src = fetchFromGitHub {
     owner = "armosec";
```

Comment:
hash needs update

Reply:
Oh woops

Reply:
![hash](https://user-images.githubusercontent.com/9866621/174391009-c81f86ec-2019-4b00-b001-65e704790be4.png)

:eyes: wtf

it's the same as 2.0.156

Reply:
ok, 157 adds a line and 158 deletes that line

Reply:
I did not expect that....